tests/x86/legacyjit_x86__minopts_testenv.cmd

```diff
@@ -0,0 +1,12 @@
+@REM -------------------------------------------------------------------------
+@REM 
+@REM  This script provides x86 Legacy JIT with MinOpts test environment settings
+@REM
+@REM -------------------------------------------------------------------------
+
+set COMPLUS_AltJit=*
+set COMPLUS_AltJitNgen=*
+set COMPLUS_AltJitName=legacyjit.dll
+set COMPLUS_NoGuiOnAssert=1
+set COMPLUS_AltJitAssertOnNYI=1
+set COMPlus_JITMinOpts=1
```

Comment:
I think the better option instead of creating many different test environments with different stress scenarios is to  load a base testenv then add stress modes. It is unclear to me how our performance infrastructure is extensible to stress scenarios like min opts as it is today.